lua/telescope/make_entry.lua

```diff
@@ -158,14 +158,26 @@ do
     mt_file_entry.cwd = cwd
     mt_file_entry.display = function(entry)
       local hl_group, icon
-      local display = utils.transform_path(opts, entry.value)
+      local display, path_style = utils.transform_path(opts, entry.value)
 
       display, hl_group, icon = utils.transform_devicons(entry.value, display, disable_devicons)
 
+      local function get_offset()
+        local offset = 0
+
+        if hl_group then
+          offset = #icon + 1
+        end
+
+        return offset
+      end
+
       if hl_group then
-        return display, { { { 0, #icon }, hl_group } }
+        local style = { { { 0, get_offset() }, hl_group } }
+        style = utils.merge_styles(style, path_style, get_offset())
```

Comment:
Is this `get_offset` function necessary? Seems redundant to me. Something similar to what been done to `gen_from_vimgrep` should work no?

Reply:
You are correct. I cleaned it up and pushed my changes.